website/project/model.py

```diff
@@ -1317,6 +1337,26 @@ def has_addon_on_children(self, addon):
 
         return False
 
+    def merge_public_files(self, node):
+
+        if not self.is_public_files_collection:
+            raise NodeStateError('must be Public Files collection to merge')
+
+        from website.files.models.osfstorage import OsfStorageFile
+
+        primaryUserFiles = OsfStorageFile.find(Q('node', 'eq', self) & Q('title', 'ne', 'Public Files'))
```

Comment:
Prefer `underscore_method` to `camelCase` for variables, as specified in the [dev docs](http://cosdev.readthedocs.io/en/latest/style_guides/python.html)